quarterback-packages/db/prisma/schema.prisma

```diff
@@ -15,6 +15,7 @@ model ManuscriptDoc {
   manuscript_model_id String               @id @default(uuid())
   user_model_id       String
   project_model_id    String
+  version             Int
```

Comment:
@mbetamony we need to change db schema to save the version. The version is always int. It is produced by frontend and based on steps count.